floris/simulation/turbine.py

```diff
@@ -265,6 +265,8 @@ def power(
     for turb_type in turb_types:
         # Using a masked array, apply the thrust coefficient for all turbines of the current
         # type to the main thrust coefficient array
+        if (rotor_effective_velocities < 0.).any():
```

Comment:
This could be moved outside of the for-loop

Reply:
Since `rotor_effective_velocities` is an input argument to this routine, this warning could also be shown via `FlorisInterface.get_turbine_powers()`. I propose to move it there to keep the input validation in the high-level code.

Reply:
@rafmudaf Good suggestion, thanks. I've updated that.